ecosystem/sep-0006.md

```diff
@@ -536,7 +536,7 @@ Name | Type | Description
 `destination_asset` | string | The code of the on-chain asset the user wants to get from the Anchor after doing an off-chain deposit. The value passed must match one of the codes listed in the [/info](#info) response's `deposit-exchange` object.
 `source_asset` | string | The off-chain asset the Anchor will receive from the user. The value must match one of the `asset` values included in a [`SEP-38 GET /prices?buy_asset=stellar:<destination_asset>:<asset_issuer>`](sep-0038.md#get-prices) response using [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
 `quote_id` | string | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this parameter is provided and the user delivers the deposit funds to the Anchor before the quote expiration, the Anchor should respect the conversion rate agreed in that quote. If the values of `destination_asset`, `source_asset` and `amount` conflict with the ones used to create the [SEP-38] quote, this request should be rejected with a `400`.
-`amount` | string | The amount of the source_asset the user would like to deposit to the anchor's off-chain account. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
+`amount` | string | The amount of the 1source_asset1 the user would like to deposit to the anchor's off-chain account. This field may be necessary for the anchor to determine what KYC information is necessary to collect. If `quote_id` was used, `amount` should be equals the quote's `sell_amount`.
```

Comment:
Is `1source_asset1` typo?

Reply:
Fixed typo in https://github.com/stellar/stellar-protocol/pull/1199/commits/e963f0687d3ee25e7f62ec29db9b73ea584c2de6. Thanks for pointing that out.